flink-connector/src/main/java/com/datastrato/gravitino/flink/connector/hive/GravitinoHiveCatalogFactoryOptions.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.flink.connector.hive;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+import org.apache.hadoop.hive.conf.HiveConf;
+
+public class GravitinoHiveCatalogFactoryOptions {
+
+  /** Identifier for the {@link GravitinoHiveCatalog}. */
+  public static final String IDENTIFIER = "gravitino-hive";
```

Comment:
move the identifier to `XXOptions`. It's the same as the HiveCatalogFactory in Flink.

Reply:
ok